pkg/policy/selectorcache.go

```diff
@@ -197,6 +209,10 @@ type SelectorCache struct {
 	selectors map[string]identitySelector
 
 	localIdentityNotifier identityNotifier
+
+	userCond  *sync.Cond
+	userMutex lock.Mutex
+	userNotes []userNotification
```

Comment:
It would be nice to have some comments about these fields.

Reply:
Added :-)